Cargo.toml

```diff
@@ -31,6 +31,7 @@ std = ["crypto/std", "lang_items/std", "persistent_store/std"]
 verbose = ["debug_ctap", "libtock_drivers/verbose_usb"]
 with_ctap1 = ["crypto/with_ctap1"]
 with_nfc = ["libtock_drivers/with_nfc"]
+with_vendor_hid = []
```

Comment:
Let's not propagate this [bad style](https://rust-lang.github.io/api-guidelines/naming.html#feature-names-are-free-of-placeholder-words-c-feature) and name the feature `vendor_hid`.

Reply:
I only renamed the new feature. Also added the flag for deploy, even though it's not useful yet.